macos/file_picker/Sources/file_picker/FilePickerPlugin.swift

```diff
@@ -45,28 +52,32 @@ public class FilePickerPlugin: NSObject, FlutterPlugin {
     let extensions = args["allowedExtensions"] as? [String] ?? []
     applyExtensions(dialog, extensions)
 
-    if dialog.runModal() == NSApplication.ModalResponse.OK {
-      if allowMultiple {
-        let pathResult = dialog.urls
-        if pathResult.isEmpty {
-          result(nil)
+    if let appWindow = getFlutterWindow() {
```

Comment:
Can we use `guard let` for the `getFlutterWindow()` calls, with a `result(nil)` ? Then we can keep the diff pretty and the code will be more readable :)